apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.ts

```diff
@@ -437,6 +465,11 @@ export class ListDetailsPanelComponent implements OnChanges, OnInit {
     });
   }
 
+  public copyTextExport() {
+    if (this._clipboardService.copyFromContent(this.getTextExport()))
+      this.textCopied();
+  }
```

Comment:
Using the [cdk directive](https://material.angular.io/cdk/clipboard/overview) here would be better, as it's doing exactly this but it's maintained without us having to do anything.